torch/csrc/jit/autodiff.cpp

```diff
@@ -389,7 +402,10 @@ static std::vector<Value*> gradientForNode(Node* node, ArrayRef<Value*> grad_val
       }
     } else if (comparison_ops.find(node)) {
       return {nullptr, nullptr};
-
+    
+    } else if (node->matches("aten::rand_like(Tensor self) -> Tensor")) {
+      return {nullptr};
```

Comment:
I’m not 100% sure that this won’t lead us to some pretty weird errors in AD later. I’ll need to check that later today, so please don’t merge yet.